### DISCUSSION TITLE
issue: resetField doesn't work for useFieldArray with no registered fields

### DISCUSSION THREAD

**Discussion by alfrsvn** [react-hook-form/react-hook-form#8299] [Q&A]
### Version Number

7.30.0

### Codesandbox/Expo snack

https://codesandbox.io/s/elastic-mahavira-fz0xy6

### Steps to reproduce

1. Go to sandbox.
2. Add some items.
3. Press resetField.
4. List is not reset.

### Expected behaviour

Field array should be reset with the defaultValue provided in useForm.

It only works if you register a field inside the field array:

```register(`listPhone.${index}.number`))```

Workaround:

`setValue('listPhone', [])`

### What browsers are you seeing the problem on?

Firefox, Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
Expected behaviour and here are some of the things you should consider to change

1. `useFieldArray` need an array-like name, that's what's been written in the doc
2. it's recommended to provide a reset value if you can.
3. you have two useForm hook, they are separate instance and not connected to each other, you can consider to use form context

**Reply by alfrsvn** [reply]
1. Is `name: 'listPhone'` wrong?
2. Noted.
3. Yes, the second one is used as input to the field array.

Is there a way to get the current defaultValue of a field?
I'm trying to avoid passing around the defaultValue to do `setValue('listPhone', defaultValue)` for this scenario.
